Makefile

```diff
@@ -781,10 +797,20 @@ COMMON_SRC = \
             io/vtx_string.c \
             io/vtx_smartaudio.c \
             io/vtx_tramp.c \
-            io/vtx.c \
+            io/vtx.c
+            
+COMMON_DEVICE_SRC = \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC)
 
+ifeq ($(OSD_SLAVE),yes)
+TARGET_FLAGS := -DUSE_OSD_SLAVE $(TARGET_FLAGS)
+COMMON_SRC := $(COMMON_SRC) $(OSD_SLAVE_SRC) $(COMMON_DEVICE_SRC)
+else
+TARGET_FLAGS := -DUSE_FC $(TARGET_FLAGS)
```

Comment:
@ledvinap , the trouble is `USE_FC` doesn't mean "use FC" it means "not osd slave"

"What happens when other non-fc variants are added? It can get out of hand quickly ..." - well we already have another non-fc target, the simulator, and it's build flags are somewhat orthogonal to the osd slave issue.

And I certainly don't favour `USE_NON_FC` since this will result in double-negatives all over the place.

I think the correct thing to do is to use `#ifdef USE_OSD_SLAVE` and `#ifndef USE_OSD_SLAVE` as appropriate.